Resources/Maps/Shuttles/DeltaV/NTES_Right.yml

```diff
@@ -7571,7 +7444,10 @@ entities:
       pos: -7.5,-18.5
       parent: 1
     - type: RandomSpawner
+      prototypes: []
```

Comment:
Can you remap this salvage canister spawner whole your doing this too, please. I don't know why all this extra stuff is on it but I don't like it.

Reply:
Okidoki